### PR TITLE
IfcSpatialZone PDTs added

### LINKED ISSUE #387: [IFC Tunnel] New PredefinedTypes for IfcSpatialZone
See 
#358
#360
#366 
#372
Definitions may need some love...
- MAPPEDZONE : A spatial zone used for collecting and recording observations of different kinds.
- TESTEDZONE : A limited zone subjected to in situ tests. Space to carry the related information on test results, methodology and other metadata. Used similar to sample, but for In-situ test
- COMPARTMENT : A tunnel compartment achieved by cut-offs  
- ANNULARGAP : Clearance between excavated surface and lining in tunneling
- CLEARANCE : <<ToDo: Definition>>
- INSTALLATION : Space used for installations, e.g. electrical, ventilation
- INTERIOR : Available space in a tunnel when lining has been installed
- INVERT : The lowest section of a tunnel (floor space)
- LINING : The space in a tunnel used for lining
- SERVICE : Space for e.g. a service gallery in a tunnel

### PR DESCRIPTION
**Pull request by larswik**
Fixes #387.
Definitions needs to be reviewed and maybe generalized!

### CODE REVIEW

File: IFC4x3/Constants/i/INVERT_3ak1q6i1jEY9NYIl0zD_62.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="INVERT_3ak1q6i1jEY9NYIl0zD_62" Name="INVERT" UniqueId="e4b81d06-b01b-4e88-95e2-4af03d37e182">
+	<Documentation>The lowest section of a tunnel (floor space)</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Constants/i/INVERT_3ak1q6i1jEY9NYIl0zD_62.xml]:
```suggestion
	<Documentation>The lowest section of a tunnel (floor space).</Documentation>
```

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Types/IfcSpatialZoneTypeEnum/DocEnumeration.xml [hunk 2]
```diff
@@ -17,6 +17,16 @@
 		<DocConstant xsi:nil="true" href="INTERFERENCE_1YJr7LhNL3efA_mK43uKGl" />
 		<DocConstant xsi:nil="true" href="USERDEFINED_2jovVwvnrBkgc5ik9l6kMP" />
 		<DocConstant xsi:nil="true" href="NOTDEFINED_3dRd_uKBHD4ebyIpTBlZOm" />
+		<DocConstant xsi:nil="true" href="MAPPEDZONE_0jgnCtnpD0JQYvromoT_1" />
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Types/IfcSpatialZoneTypeEnum/DocEnumeration.xml:18]: Move above USERDEFINED.

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Types/IfcSpatialZoneTypeEnum/DocEnumeration.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Types/IfcSpatialZoneTypeEnum/DocEnumeration.xml:18]: Done.

File: IFC4x3/Constants/c/CLEARANCE_2WTa08IAn2wQFVvRSpIrPs.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CLEARANCE_2WTa08IAn2wQFVvRSpIrPs" Name="CLEARANCE" UniqueId="a0764008-48ac-42e9-a3df-e5b7334b5676">
+	<Documentation>The space required for the safe operation of vehicles such as a locomotive along a fixed track,</Documentation>
```
Comment (anborr) [inline IFC4x3/Constants/c/CLEARANCE_2WTa08IAn2wQFVvRSpIrPs.xml]:
```suggestion
	<Documentation>The space required for the safe operation of vehicles such as trains along a fixed track.</Documentation>
```

**Review by anborr — CHANGES_REQUESTED** [review]

**Review by SergejMuhic — APPROVED** [review]

**Comment by SergejMuhic** [comment]
Looks as though @anborr comments were addressed.

**Pull request merged**